examples/aws/auto-scaling-group/auto-scaling-group.tf

```diff
@@ -0,0 +1,26 @@
+# --- RESOURCES / MODULES ---
+
+data "aws_availability_zones" "azs" {
+  state = "available"
+}
```

Comment:
technically this should be inside a data block 🙈 

```suggestion
# --- DATA ---

data "aws_availability_zones" "azs" {
  state = "available"
}

# --- RESOURCES / MODULES ---
```

Reply:
Yes, true! :D